src/components/Hoverable/index.js

```diff
@@ -69,6 +69,9 @@ class Hoverable extends Component {
      * @param {Object} event - A click event
      */
     resetHoverStateOnOutsideClick(event) {
+        if (!this.state.isHovered) {
```

Comment:
The hoverable component is a very basic component and it is used in many places. We should prevent re-render at all costs. Here Every click basically causes a re-render for all child components. But in reality, we only want to `resetHoverStateOnOutsideClick` when we were actually hovering over a component. We utilize this feature on Context Menu and we want to reset hover state after menu close this `mousedown`.